tests/conftest.py

```diff
@@ -37,7 +37,6 @@ def model_name(request: pytest.FixtureRequest):
 
 
 @pytest.fixture(
-    autouse=True,
```

Comment:
Why was this necessary ?

Reply:
`autouse` makes all tests use the fixture, even when not requested, we added it at the beginning thinking that all tests would require this, but it's actually only needed for some tests, thus we can relax this.